PWGHF/D2H/TableProducer/candidateSelectorB0ToDPiReduced.cxx

```diff
@@ -134,6 +149,15 @@ struct HfCandidateSelectorB0ToDPiReduced {
         // LOGF(info, "B0 candidate selection failed at topology selection");
         continue;
       }
+
+      if constexpr (withDmesMl) { // we include it in the topological selections
```

Comment:
I have no definite answer here because it depends how tight are the two different cuts: for sure if we cut a lot with the D-meson ML scores, it is convenient to put them upfront (especially if the other cuts require computations, since the ML scores at this stage don't require any calculation), otherwise probably it does not change much

Reply:
Okay, thanks :)